examples/mfa.md

```
@@ -0,0 +1,474 @@
+
+The MFA API allows you to manage multi-factor authentication for users. The SDK automatically handles MFA context, eliminating the need for manual parsing of error payloads.
+
+> [!NOTE]
+> Multi Factor Authentication support via SDKs is currently in Early Access. To request access to this feature, contact your Auth0 representative.
+
+- [Understanding the MFA Response](#understanding-the-mfa-response)
+- [Handling MFA required errors](#handling-mfa-required-errors)
+- [Getting Authenticators](#getting-authenticators)
+- [Getting Enrollment Factors](#getting-enrollment-factors)
+- [Enrollment](#enrollment)
+  - [Enrolling OTP (Authenticator App)](#enrolling-otp-authenticator-app)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)